examples/Makefile.am

```diff
@@ -45,8 +43,10 @@ EXAMPLES				= hildon-window-example				\
 					  hildon-remote-texture-example			\
 					  hildon-gtk-window-take-screenshot-sync
 if USE_MAEMO_GTK
-MAEMO_GTK_EXAMPLES                      = hildon-pannable-area-touch-list-example	\
-					  hildon-pannable-area-touch-grid-example
+MAEMO_GTK_EXAMPLES  = hildon-pannable-area-touch-list-example	\
+					  hildon-pannable-area-touch-grid-example \
+					  hildon-live-search-example			\
+					  hildon-live-search-grid-example
```

Comment:
Changing tab/spaces count here makes code look inconsistent. Please fix.

Reply:
I didn't change to spaces. It's neatly aligned with 4 spaces tab.